samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -108,8 +108,8 @@ static struct bt_conn *broadcast_assistant_conn;
 static struct bt_le_ext_adv *ext_adv;
 
 static const struct bt_audio_codec_cap codec_cap = BT_AUDIO_CODEC_CAP_LC3(
-	BT_AUDIO_CODEC_LC3_FREQ_16KHZ | BT_AUDIO_CODEC_LC3_FREQ_24KHZ,
-	BT_AUDIO_CODEC_LC3_DURATION_10, BT_AUDIO_CODEC_LC3_CHAN_COUNT_SUPPORT(1), 40u, 60u, 1u,
+	BT_AUDIO_CODEC_CAP_FREQ_16KHZ | BT_AUDIO_CODEC_CAP_FREQ_24KHZ,
```

Comment:
looks like a breaking change or am I missing something? shouldn't the `...LC3...` option only be deprecated for now?
and would need entires in release notes / migration guide too

Reply:
@kartben My understanding is that for experimental APIs (like LE Audio), this is not necessary. 

Given how long time these macros have existed, it may make sense to put something in the migration guide, but AFAIK that's not something we've done before for experimental APIs. 

What do you think?

Reply:
Ah, forgot that LE Audio was experimental, sorry. So yes, you're right, it's not strictly necessary. If you don't mind, and based on your comment re: how long they've been around, a short note in the migration guide would be welcome indeed. Thanks!